cachet-tools/cachet-updater-secret.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: cachet-updater-secret 
+data:
+  cachethostnamekey: YW0tY2FjaGV0LmFwcHMudGFjb3N1cHJlbWUub3V0c3JpZ2h0cy5jYw== 
```

Comment:
Can you sanitize this secrets file?  Don’t want actual secrets in here.

Reply:
Oh, I didn't mean to check this file. I created this yaml to not have manually create secrets.  At the time and even now I am not 100% sure how to deal with it.  In the template for AppSRE, I plan to create a secret this secret based on the parameter values passed in.